XCUITests/NewTabSettings.swift

```diff
@@ -30,6 +30,23 @@ class NewTabSettingsTest: BaseTestCase {
         waitForNoExistence(app.collectionViews.cells["TopSitesCell"].collectionViews.cells["youtube"])
         waitForNoExistence(app.staticTexts["Highlights"])
     }
+    
+    func testChangeNewTabSettingsShowFirefoxHome() {
+        // Set to blank page first since FF Home is default
+        navigator.goto(NewTabSettings)
+        navigator.performAction(Action.SelectNewTabAsBlankPage)
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        waitForNoExistence(app.collectionViews.cells["TopSitesCell"])
+        
+        // Now check if it switches to FF Home
+        navigator.goto(SettingsScreen)
+        navigator.goto(NewTabSettings)
+        navigator.performAction(Action.SelectNewTabAsFirefoxHomePage)
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        waitForExistence(app.collectionViews.cells["TopSitesCell"])
+        waitForExistence(app.collectionViews.cells["TopSitesCell"].collectionViews.cells["youtube"])
```

Comment:
I would say this and the next wait are redundant, if the TopSitesCell appears then it would be fine...